workflows/flows/analysis/assembly/tasks/add_assembly_study_summaries_to_downloads.py

```diff
@@ -0,0 +1,111 @@
+from pathlib import Path
+
+from prefect import task, get_run_logger
+
+import django
+
+django.setup()
```

Comment:
I kind of prefer to use:

```suggestion
import activate_django_first  # noqa
```

just because then we have one place to add any other setup magic that may become required in future, that has historically only become a problem in prod

Reply:
yeah, let's keep it consistent